ember.js/2018-12/F2F-Ember-Core-Team-20181215.md

```diff
@@ -0,0 +1,312 @@
+# Notes from F2F, Portland, Dec 2018
+
+## Agenda
+
+F2F Agenda (covering both days):
+
+- Proposal: Internal Communication Strategy 
+- Proposal: Core Team Member Roles and Responsibilities (Process Document)
+- `{{use` / Template Imports / Module Unification Namespacing [RJ]
+  - https://github.com/emberjs/rfcs/pull/367
+  - https://github.com/ef4/rfcs/pull/16
+- Upcoming F2F’s [RJ]
+  - Timelines
+  - Weekdays?
+- TC39 style stage system for RFCs [RJ]
+- Accessible Routing RFC (Work in progress, not submitted yet) [MS]
+- Specific process to change the default application / addon blueprints. [RJ]
+- Octane-related RFCs:
+  - Editions
+  - Remove jQuery by Default
+  - Glimmer Components
+  - Rendering Element Modifiers
+  - Tracked Properties
+  - Object Model Decorators
+- Website re-design preview/discussion
+- Conversation: Technical Debt 
+- Are we happy to continue with all core teams are equal core teams? [KG]
```

Comment:
I think this sounds bad taken out the context 😬 
We also did not end up discussing this.